.buildkite/scripts/steps/scalability/api_capacity_testing.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+source .buildkite/scripts/steps/scalability/util.sh
+
+#.buildkite/scripts/bootstrap.sh
+echo "--- yarn kbn reset && yarn kbn bootstrap"
+yarn kbn reset && yarn kbn bootstrap
```

Comment:
any reason we're using this instead of `.buildkite/scripts/bootstrap.sh`?  we probably want to pull as much from cache as possible

Reply:
I tried it few time and since it is bare metal machine, I think it misses access to cache

<img width="1166" alt="image" src="https://user-images.githubusercontent.com/10977896/220967868-9b4af79a-7cf3-417a-9f2d-439051bc86bc.png">

Maybe you know what I should do here?

Reply:
We can hold off for now - it looks like the bootstrap script expects a google cloud vm.  Thanks for taking a look.